src/include/connection.h

```diff
@@ -29,6 +29,7 @@ struct __wt_process {
 
     bool fast_truncate_2022; /* fast-truncate fix run-time configuration */
     bool page_stats_2022;    /* Page stats run-time configuration */
+    bool tiered_shared_2023; /* tiered shared run-time configuration */
```

Comment:
Why not name it union_table_2022?

Reply:
Yes, it can be, I thought the project would start in 2023 and marked it as 2023 in the design doc. I used the same name as in the design doc to keep them identical.